### DISCUSSION TITLE
Any possible fix for the alertmanager generatorURL breaking due to url-encoding issues

### DISCUSSION THREAD

**Discussion by aditya-opsverse** [grafana/grafana#54614] [Alerting]
I have set-up `alertmanager` and configured it using the [official documentation](https://prometheus.io/docs/alerting/latest/configuration/). The set-up is made to send alerts to Slack, Email and Telegram via the `alertmanager`.
I have triggered some alerts to see if it is working fine. For Slack and Emails, the `generatorURL` is clickable and when clicked takes back to the `Grafana Explore` section which is the expected behaviour. However for Telegram, the `generatorURL` link is broken (The link is half clickable).

After some research, I found out that it is because of the url-encoding (i.e. The quotes, square brackets, etc. are not encoded). I don't know how to encode it manually. However I have tried two solutions:

1. Used `parseMode` as `HTML` and using anchor tag for the `generatorURL` - Didn't work.
2. Used `reReplaceAll` to replace the quotes, square brackets, etc. into url encoded form from this [GitHub Discussion](https://github.com/grafana/grafana/discussions/42316) - Didn't work.
Any suggestions on what can be done to resolve this issue?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
